volttron/platform/auth.py

```diff
@@ -137,6 +137,9 @@ def read_auth_file(self):
         if self._is_connected:
             try:
                 _log.debug("Sending auth updates to peers")
+                # Give it few seconds for platform to startup or for the
+                # router to detect agent install/remove action
+                gevent.sleep(5)
```

Comment:
I think that we shouldn't do this as every time something triggers an auth read we will be sleeping for an arbitrary 5 seconds.

Reply:
It is needed to detect new installed/removed agents. Perhaps I can reduce the sleep period